### ISSUE TITLE
维度不匹配

### ISSUE THREAD

**Issue by VOGGD** [jdw-art/stdp_classification#2]
作者你好，在调用STDPClassifyExcInhNet网络的时候，由于批次未设置为1，导致这里 # STDP
delta_hidden_weight = self.plasticity_rule(l1_trace, l2_trace, self.hidden_weight.unsqueeze(0)).detach()
with torch.no_grad():
self.hidden_weight += delta_hidden_weight.squeeze(0)

            # 更新兴奋-抑制层权重
            inhibitory_trace = exp_convolve(inhibitory, inhibitory_trace, self.decay_trace).detach()
            delta_exc_inh_weight = self.plasticity_rule(l2_trace, inhibitory_trace, self.exc_inh_weight.unsqueeze(0)).detach()
            delta_inh_exc_weight = self.plasticity_rule(inhibitory_trace, l2_trace, self.inh_exc_weight.unsqueeze(0)).detach()
            with torch.no_grad():
                self.exc_inh_weight += delta_exc_inh_weight.squeeze(0)
                self.inh_exc_weight += delta_inh_exc_weight.squeeze(0)

是不是会出问题，比如此时delta_exc_inh_weight.squeeze(0)中squeeze(0)是无效的。导致self.inh_exc_weight和delta_inh_exc_weight.squeeze(0)维度不匹配。想请问作者是不是要把批次设置为1，还是要把delta_inh_exc_weight/批次，将维度与self.inh_exc_weight对应上？
